userservices/client/standard.go

```diff
@@ -218,16 +218,16 @@ func (s *Standard) GetUserGroupID(context service.Context, userID string) (strin
 // AND bad session token. Since a bad server token is unlikely (though possible) we MUST assume
 // that it means bad session token.
 
-func (s *Standard) sendRequest(context service.Context, method string, url string, responseObject interface{}) error {
+func (s *Standard) sendRequest(context service.Context, requestMethod string, requestURL string, responseObject interface{}) error {
```

Comment:
Hmm, trying to remember. I believe it was for consistency sake with another sendRequest is either the metricservices or dataservices client.